docs/ops.html

```diff
@@ -787,18 +787,18 @@ <h5 class="anchor-heading"><a id="georeplication-flow-configure" class="anchor-l
 
   <ul>
     <li><code>refresh.topics.enabled</code>: whether to check for new topics in the source cluster periodically (default: true)
-    <li><code>refresh.topics.interval.seconds</code>: frequency of checking for new topics in the source cluster; lower values than the default may lead to performance degradation (default: 6000, every ten minutes)
+    <li><code>refresh.topics.interval.seconds</code>: frequency of checking for new topics in the source cluster; lower values than the default may lead to performance degradation (default: 600, every ten minutes)
     <li><code>refresh.groups.enabled</code>: whether to check for new consumer groups in the source cluster periodically (default: true)
-    <li><code>refresh.groups.interval.seconds</code>: frequency of checking for new consumer groups in the source cluster; lower values than the default may lead to performance degradation (default: 6000, every ten minutes)
+    <li><code>refresh.groups.interval.seconds</code>: frequency of checking for new consumer groups in the source cluster; lower values than the default may lead to performance degradation (default: 600, every ten minutes)
     <li><code>sync.topic.configs.enabled</code>: whether to replicate topic configurations from the source cluster (default: true)
     <li><code>sync.topic.acls.enabled</code>: whether to sync ACLs from the source cluster (default: true)
     <li><code>emit.heartbeats.enabled</code>: whether to emit heartbeats periodically (default: true)
-    <li><code>emit.heartbeats.interval.seconds</code>: frequency at which heartbeats are emitted (default: 5, every five seconds)
+    <li><code>emit.heartbeats.interval.seconds</code>: frequency at which heartbeats are emitted (default: 1, every one seconds)
```

Comment:
default should be 1 second
https://github.com/apache/kafka/blob/trunk/connect/mirror/src/main/java/org/apache/kafka/connect/mirror/MirrorConnectorConfig.java#L173